pkg/client/unversioned/testclient/simple/simple_testclient.go

```diff
@@ -53,12 +54,13 @@ type Response struct {
 
 type Client struct {
 	*client.Client
-	Request  Request
-	Response Response
-	Error    bool
-	Created  bool
-	server   *httptest.Server
-	handler  *utiltesting.FakeHandler
+	Clientset *release_1_1.Clientset
```

Comment:
In theory, the release 1.1 client set would have been generated and locked months ago. We should be working on the release 1.2 client set right now, and we should lock it in place a week from now, and then start working on the release 1.3 client set.